src/containers/main-page/stepper/StepperControls.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Button from '@mui/material/Button';
+import Box from '@mui/material/Box';
+
+interface StepperControlsProps {
+    activeStep: number;
+    handleBack: () => void;
+    handleNext: () => void;
+    stepsLength: number;
+}
+
+const StepperControls = function ({
+  activeStep,
+  stepsLength,
+  handleBack,
+  handleNext,
+}: StepperControlsProps): JSX.Element {
+  return (
+    <Box sx={{ display: 'flex', flexDirection: 'row', pt: 2 }}>
+      <Button
+        disabled={activeStep === 0}
+        onClick={handleBack}
+        sx={{ mr: 1 }}
+        variant="contained"
+        color="secondary"
+      >
+        Back
+      </Button>
+      <Box sx={{ flex: '1 1 auto' }} />
```

Comment:
Also better to replace into the StyledComponent. Then would be better to find and refactor styles if it needs.